### DISCUSSION TITLE
MAKE IT CLEARER NOT TO MENTION ON SCRATCH

### DISCUSSION THREAD

**Discussion by Mbrick2** [ScratchAddons/ScratchAddons#3649] [Discussion]
I have seen people even [post the link](https://scratch.mit.edu/discuss/topic/550055/?page=1#post-5693501) on the forums. I really think when you open settings a popup should come up saying

> Please remember to not mention the extension on Scratch. To know why click here (link to the post about it)

**Comment by Secret-chest** [comment]
And it should have an X button

**Comment by Samq64** [comment]
This is already implemented in some places.

![image](https://user-images.githubusercontent.com/81489795/137159917-ce91c7f6-2cf1-4320-8f73-4c4756b44e0b.png)

![image](https://user-images.githubusercontent.com/81489795/137159745-8e07938f-aac3-4461-98e8-ca06191f9e67.png)

**Reply by WorldLanguages** [reply]
Keep in mind that the rate in which users advertise us on the forums isn't very high, according to ScratchDB. But something like what we have for comments could be implemented in the forums.

**Reply by Mbrick2** [reply]
> Keep in mind that the rate in which users advertise us on the forums isn't very high, according to ScratchDB. But something like what we have for comments could be implemented in the forums.

Whats the amount??

**Reply by WorldLanguages** [reply]
Well, search for "scratchaddons.com" and "scratch addons" on ocular.jeffalo.net sorting by newest, and you won't see more than 15 posts per month from my experience.

**Reply by Mbrick2** [reply]
Thanks
